src/patterns/ec2-app/base.ts

```diff
@@ -377,6 +449,115 @@ export class GuEc2App extends Construct {
       }
     }
 
+    if (props.googleAuth?.enabled) {
+      const prefix = `/${scope.stage}/${scope.stack}/${app}`;
+
+      const {
+        allowedGroups = ["engineering@theguardian.com"],
+        clientIDSSMPath = `${prefix}/google-auth-client-id`,
+        clientSecretSecretsManagerPath = `${prefix}/google-auth-client-secret`,
```

Comment:
I wonder if we should include the pattern here - i.e. `ec2-google-auth-client-id` - just in case we introduce Google Auth support for e.g. API Gateway too in future.

Reply:
(In fact, this is not relevant, as we already include the app name here which should be unique.)